src/main/java/org/usfirst/frc4904/robot/humaninterface/operators/DefaultOperator.java

```diff
@@ -14,5 +15,7 @@ public DefaultOperator(String name) {
 	}
 
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		RobotMap.HumanInput.Operator.joystick.button1.onlyWhileHeld(new ClimbDistance()); //Use climb with distance (ClimbDistance) or climb with time (ClimbTime)
```

Comment:
make this a TODO, and I'd start with `ClimbTime()` (because it's gonna be more reliable